src/pybind/mgr/dashboard/controllers/osd.py

```diff
@@ -168,9 +168,11 @@ def safe_to_destroy(self, svc_id):
             svc_id = [svc_id]
         svc_id = list(map(str, svc_id))
         try:
-            CephService.send_command(
+            result = CephService.send_command(
                 'mon', 'osd safe-to-destroy', ids=svc_id, target=('mgr', ''))
-            return {'safe-to-destroy': True}
+            return {'safe-to-destroy': set(result['safe_to_destroy']) ==
+                    set(map(lambda id: int(id), svc_id)),
+                    'details': result}
```

Comment:
do we really need to put that into a sub object `details`? Another idea would be to just unify them:
```python
result['safe-to-destroy'] = ...
return result
```

Reply:
No, we don't need that, it's just for differentiating the computed output from the one that's just passed through.

Reply:
ah, we're returning something like
```javascript
{
    'safe-to-destroy': true,
    'details': {
        'safe_to_destroy': [1,2,3],
        ...
    }
}
```

Which is strange from an API design POV. Especially as the dashboard will be the official API at some point in the future.

Reply:
Can you please be more specific about how you'd like to have this implemented? Do you think the computed `true|false` values shouldn't be returned by the backend? If you think it's okay to provide it, how would you name is so that the keys don't collide, when the result is just a single object as you've requested?

Reply:
What do you think of something like this?
```javascript
{
    'is_safe_to_destroy': true,
    'safe_to_destroy': [1,2,3],
    ...
}
```

Reply:
http://tracker.ceph.com/issues/37327